lib/util.js

```diff
@@ -81,8 +92,8 @@ module.exports = (function() {
         return numDevices;
     }
 
-    function getSimId() {
-        var findSimCommand = "cordova run --list --emulator | grep ^iPhone | tail -n1";
+    function getSimId(localCLI) {
```

Comment:
Can't we get the localCLI here by just calling getlocalCLI() instead of having to pass it to updatepermissions and then to getSimId?